VAMobile/src/screens/BenefitsScreen/ClaimsScreen/AppealDetailsScreen/AppealStatus/AppealCurrentStatus/AppealCurrentStatus.test.tsx

```diff
@@ -44,18 +43,6 @@ context('AppealStatus', () => {
     component = render(<AppealCurrentStatus {...props} />, {
       preloadedState: {
         ...InitialState,
-        personalInformation: {
-          ...InitialState.personalInformation,
-          profile: {
-            firstName: '',
-            middleName: '',
-            lastName: '',
-            signinEmail: '',
-            birthDate: '',
-            fullName: 'Larry Brown',
-            signinService: 'IDME',
-          },
-        },
       },
     })
```

Comment:
The `preloadedState` param can be removed also, since we're not setting the state anymore

Reply:
Looks like this one was missed